docs/glossary.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Glossary
+---
+
+## Terms
+
+### button
+#### See Also
+*   [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)
+
+### component
+A small, reusable piece of code that that returns an element to render.
+#### See Also
+*   [https://reactjs.org/docs/glossary.html#components](https://reactjs.org/docs/glossary.html#components)
+
+### computed property
+A property whose value is bound to that of one or more dependent properties.
+
+### Date
+#### See Also
+*   [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)
+
+### element
+Describes what you want to see on the screen, such as a heading DOM element with some text: `<h1>I'm an element!</h1>`.
+#### See Also
+*   [https://reactjs.org/docs/glossary.html#elements](https://reactjs.org/docs/glossary.html#elements)
+
+### event
+A message-passing strategy for indirect communication between components.
```

Comment:
I think I would update this to be more React-specific.  Can link to React docs on event, too.